### ISSUE TITLE
Proxymane doesn't handle multiple `www-authenticate` headers on response

### ISSUE THREAD

**Issue by TingluoHuang** [ProxymanApp/Proxyman#349]
### Proxyman version? (Ex. Proxyman 1.4.3)
1.10.0 (11000)
### macOS Version? (Ex. mac 10.14)
10.15.1
### Steps to reproduce 
- Close Proxyman: `curl -D - https://tingweu.visualstudio.com/_apis/distributedtask/pools`
response headers contains 2 entries for `www-authenticate`
```
www-authenticate: Bearer authorization_uri=https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47
www-authenticate: Basic realm="https://tfsprodweu2.app.visualstudio.com/"
```

- Open Proxyman: `curl -k -x http://127.0.0.1:9090 -D - https://tingweu.visualstudio.com/_apis/distributedtask/pools`
response headers only contains 1 entry for `www-authenticate`
```
WWW-Authenticate: Basic realm="https://tfsprodweu2.app.visualstudio.com/"
```

Feels like there is a bug in the code that only take the last `www-authenticate` header. 😕 

### Expected behavior

With proxyman open, response header for `www-authenticate` should not change

**Comment by NghiaTranUIT** [comment]
Hi there, It's definitely a critical bug. Thank for the report, I'm fixing it now and send you a hotfix build ASAP 👍

**Comment by NghiaTranUIT** [comment]
Hi @TingluoHuang, here is the BETA fix: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_1.10.0_Missing_Header_Keys.dmg

It should work and we will officially release in the next build 👍 

Thank u for the bug report 🐛

**Comment by TingluoHuang** [comment]
Confirm the beta install fix my problem, feel free to close the issue

**Comment by NghiaTranUIT** [comment]
Good good, glad to hear that 🌮 🎄

**Comment by NghiaTranUIT** [comment]
Just friendly reminder that this bug is fixed in Proxyman 1.11.0, which we released yesterday. 

Let update it 👏

https://proxyman.io/blog/2019/06/changelogs.html
